Logger.Tests/LoggerFixture.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Logger.Tests
+{
+    public class LoggerFixture
+    {
+        private readonly ITxsLogger l = new TxsLogger();
```

Comment:
Please give valid name to ITxsLogger object instead of l

Reply:
done